vars.js

```diff
@@ -17,8 +17,8 @@ module.exports = {
   },
 
   channel: {
-    href: "https://chat.libretime.org",
-    label: "Chat",
+    href: "https://matrix.to/#/#libretime:matrix.org",
+    label: "Chat (#libretime:matrix.org)",
```

Comment:
This will be printed in the website navigation bar, so we probably want something not too long.